assets/src/dashboard/karma/fixture.js

```diff
@@ -33,6 +39,15 @@ import actPromise from '../../karma-fixture/actPromise';
 import { AppFrame } from '../components';
 import ApiProviderFixture from './apiProviderFixture';
 
+configure({
+  getElementError: (message) => {
+    const error = new Error(message);
+    error.name = 'TestingLibraryElementError';
+    error.stack = null;
+    return error;
+  },
+});
```

Comment:
```suggestion
if('true' === process.env.CI) {
configure({
  getElementError: (message) => {
    const error = new Error(message);
    error.name = 'TestingLibraryElementError';
    error.stack = null;
    return error;
  },
});
}
```

ditto for editor

Reply:
Fixed https://github.com/google/web-stories-wp/pull/6900/commits/98125256c8fdff20629173349f42f6787ed0eca3